schemas/session-replay/view-end-record-schema.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "view-end-record-schema.json",
+    "title": "ViewEndRecord",
+    "type": "object",
+    "description": "Schema of a Record which signifies that view lifecycle ended.",
+    "allOf": [
+        {
+            "$ref": "_common-record-schema.json"
+        },
+        {
+            "required": [
+                "type",
+                "data"
```

Comment:
question: is `data` defined?

Reply:
No it's not: https://github.com/DataDog/browser-sdk/blob/main/packages/rum/src/types/record.ts#L59-L62
it should be removed from here